### PR TITLE
chore: bump max_connections in ci pg

### PR DESCRIPTION
**Pull request by makramkd**

### CODE REVIEW

File: .github/workflows/ci-core.yml [hunk 1]
```diff
@@ -87,6 +71,8 @@ jobs:
         uses: ./.github/actions/setup-go
       - name: Setup Solana
         uses: ./.github/actions/setup-solana
+      - name: Setup Postgres
```
Comment (chainchad) [inline .github/workflows/ci-core.yml:74]: We should add a comment on why we're moving away from the service container to this setup.

File: .github/actions/setup-postgres/.env [hunk 2]
```diff
@@ -0,0 +1,5 @@
+POSTGRES_USER=postgres
+POSTGRES_OPTIONS="-c max_connections=1000 -c shared_buffers=2GB"
+POSTGRES_PASSWORD_=postgres
```
Comment (chainchad) [inline .github/actions/setup-postgres/.env:3]:
```suggestion
POSTGRES_PASSWORD=postgres
```
Was that trailing underscore supposed to be there?

**Review by chainchad — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-core.yml (hunk 1 above)
Comment (HenryNguyen5) [reply .github/workflows/ci-core.yml:75]: [318c864](https://github.com/smartcontractkit/chainlink/pull/7771/commits/318c864028b13e6fc7daeea857f2a949282d8a1c)

File: .github/actions/setup-postgres/.env (hunk 2 above)
Comment (HenryNguyen5) [reply .github/actions/setup-postgres/.env]: [139df8e](https://github.com/smartcontractkit/chainlink/pull/7771/commits/139df8ee5e43c19f71af03d3117a5f621f44ac44)

**Review by jmank88 — APPROVED** [review]

**Review by chainchad — APPROVED** [review]

**Pull request merged**